### DISCUSSION TITLE
Save raw or processed downloaded data to a file?

### DISCUSSION THREAD

**Discussion by alexeisenhart** [OpenBB-finance/OpenBB#493] [Q&A] [answered]
Is there any way to export the downloaded data into a file or series of files?

I'd like to be able to automate the export of the ticker prices, dd > short, dd >rating, and such so that I can analyze the data with my tools of choice. 

On a daily basis, maybe I'd run a batch job that uses the terminal to download the price, volume, shorts, etc from the previous day (for all stock tickers or at least a few that I identify as important to me), and then produce a series of CSV files which I can then consume.

Is this something that is supported, or might be in the future?

**Comment by DidierRLopes** [comment, marked as answer]
Sorry for late reply. Yes there is. We are on the works of adding this functionality. Ideally all commands will have a flag `--export` that exports the raw data in the format required. E.g. `--export csv`. There's already a couple of commands that have such implementation, more on the way.

Let us know if there's any particular command you'd want to export data that isn't already.

Regarding that "batch job" you may want to have a look to our papermill notebooks. We have been tidying them up, and they will be able to generate a report for you automatically.

**Reply by kristinWalker1991** [reply]
https://github.com/github/docs/blob/main/content/repositories/configuring-branches-and-merges-in-your-repository/defining-the-mergeability-of-pull-requests/about-protected-branches.md
